src/components/PDOKAutoSuggest/index.js

```diff
@@ -55,10 +55,13 @@ PDOKAutoSuggest.propTypes = {
   className: PropTypes.string,
   fieldList: PropTypes.arrayOf(PropTypes.string),
   /**
-   * Value that determines to which municipality the search query should be applied
-   * Can be a single name, like amsterdam, or a combination, like (amsterdam OR weesp)
+   * Value that determines to which municipality the search query should be
+   * applied.
+   *
+   * Can be a single name, like amsterdam, or an array of one or more names,
+   * which will be combined using a logical OR
    */
-  gemeentenaam: PropTypes.string,
+  municipality: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
```

Comment:
`municipality` Isn't a required prop, so it should have a default value.

Reply:
On second thought; strike that. `undefined` Is also handled properly without generating warnings.